src/mutations/item.ts

```diff
@@ -240,6 +242,12 @@ export default (queryConfig: QueryClientConfig) => {
             queryClient.invalidateQueries(parentKey);
           }
 
+          // reorder affect children to change
+          if ((extra as FolderItemExtra)?.[ItemType.FOLDER]?.childrenOrder) {
```

Comment:
Would checking the type of the item help here ?

Reply:
You cannot because you don't pass the `type` when you edit an item.